tests/pass/intrinsics-x86-avx2.rs

```diff
@@ -0,0 +1,1613 @@
+// Ignore everything except x86 and x86_64
+// Any new targets that are added to CI should be ignored here.
+// (We cannot use `cfg`-based tricks here since the `target-feature` flags below only work on x86.)
+//@ignore-target-aarch64
+//@ignore-target-arm
+//@ignore-target-avr
+//@ignore-target-s390x
+//@ignore-target-thumbv7em
+//@ignore-target-wasm32
```

Comment:
can we use `only-target-x86` instead?

Reply:
Does `only-target-x86` cover i686?

Reply:
hmno. we need https://github.com/oli-obk/ui_test/issues/162 first